.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
+    "terminal.integrated.cwd": "${workspaceFolder}",

```

Comment:
Isn't this the default?

Reply:
Yep - what's happening is that I have a user setting of `C:\Temp`, intended for when I open VS Code without a workspace. This workspace setting then restores the default.

(AFAICT, I can't restrict the user setting to apply to windows without workspaces, or I'd just do that.)